specs/vep-1739-multiple-python-versions/README.md

```diff
@@ -0,0 +1,242 @@
+
+# VEP-1739: Multiple Python Versions
+
+* **Author(s):** Andon Andonov (andonova@vmware.com)
+* **Status:** draft
+
+<!-- Provide table of content as it's helpful. -->
+
+- [Summary](#summary)
+- [Glossary](#glossary)
+- [Motivation](#motivation)
+- [Requirements and goals](#requirements-and-goals)
+- [High-level design](#high-level-design)
+- [API Design](#api-design)
+- [Detailed design](#detailed-design)
+- [Alternatives](#alternatives)
+
+## Summary
+
+---
+Currently, when a data job is deployed, the Control Service uses vdk and data job base images set once and applied to
+all job deployments. If a data engineer decides they want to use a different python version for their job, they need to
+ask their infrastructure administrator, or whoever is responsible for the Control Service deployment, to change the
+configuration of the service and re-deploy it to allow for data jobs with a different python version to be deployed.
+This, however, would break existing deployed data jobs, as the moment they are re-deployed, the new python version would
+be applied with unforeseeable consequences.
+
+We want to allow users to deploy data jobs with different python versions without needing to re-deploy the Control
+Service. To do this, we will extend the Control Service logic and API to support multiple python versions for data job
+deployments.
+
```

Comment:
Now 21-26 explains a problem, then 28-30 is the change summary. 

I would like to understand better the change from user perspective. For example, (1) the user is obliged/non-obliged to explicitly provide a python version upon deployment; (2) is a default version available, and (3) is it dynamically detected based on a local setup or team-wide preferences? (4) Are users monitoring/fetching the current deployment version set, and (5) are the version changes traceable, and (6) are the version changes related to job executions?

Would like to know the design changes proposed, are those aiming at the Control Service API or model, VDK CLI? What about backwards compatibility, that is relevant to VEP-994 in progress, that aims at UX and local development using a GUI alternative to the VDK CLI?
```
<!--
Short summary of the proposal. It will be used as user-focused
documentation such as release notes or a (customer facing) development roadmap.
The tone and content of the `Summary` section should be
useful for a wide audience.
-->
```

Reply:
Added more details to make it clearer, thank you